test/upgrade/probe_test.go

```diff
@@ -30,6 +31,10 @@ import (
 	v1test "knative.dev/serving/test/v1"
 )
 
+var (
+	successFraction = *flag.Float64("probe.success_fraction", 1.0, "Fraction of probes required to pass during upgrade.")
```

Comment:
no you should rather deref in the  `CheckSLO`.

Reply:
I'm guessing the var section gets executed before the flag parsing? Makes sense.